lte/gateway/c/oai/tasks/mme_app/mme_app_ueip_imsi_map.h

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+
+#include <bits/stdc++.h>
+#include <unordered_map>
+#include <iostream>
+#include <string>
+#include <vector>
+
+typedef std::unordered_map<std::string, std::vector<uint64_t>> UeIpImsiMap;
```

Comment:
Add comment here of what this represents, what is the key and what value it maps to